README.md

```diff
@@ -1,73 +1,33 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
 > Description the project.
+- simple HTML & CSS code, displaying Hello microverse on the broweser.
+- configure with linters 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html & Css
 
-## Live Demo (if available)
+## Author
 
-[Live Demo Link](https://livedemo.com)
+👤 **Abdulai Samuel Moiwa**
 
+- GitHub: [@githubhandle](https://github.com/samuelmoiwa)
+- Twitter: [@twitterhandle](https://twitter.com/samuelmoiwa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ing-abdulai-samuel-moiwa-726340142/)
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
```

Comment:
- [ ] I would like to recommend you add a section for `Getting started`. The easiest way to fill out this section is to help someone who has never used your project get set up, and write down the exact instructions. :+1: